src/vs/editor/contrib/rename/browser/renameWidget.ts

```diff
@@ -318,7 +318,8 @@ export class RenameWidget implements IRenameWidget, IContentWidget, IDisposable
 	}
 
 	afterRender(position: ContentWidgetPositionPreference | null): void {
-		this._trace('invoking afterRender, position: ', position ? 'not null' : 'null');
+		// @ulugbekna: commenting trace log out until we start unmounting the widget from editor properly
+		// this._trace('invoking afterRender, position: ', position ? 'not null' : 'null');
```

Comment:
It's a little messy to leave personal notes around like this. If anything it should be `// TODO:` or `// TODO@ulugbekna`, maybe with an associated issue so it's easier to find and doesn't get forgotten.